src/components/mapPageComponents/mapContainer/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect } from 'react';
+import { connect, useDispatch } from 'react-redux';
+import { AsyncRequestKinds } from '../../../utils/asyncRequest';
+import { getMapGeoData } from './ducks/thunks';
+import {
+  BlockGeoDataReducerState,
+  NeighborhoodGeoDataReducerState,
+} from './ducks/types';
+import { C4CState } from '../../../store';
+import MapView from '../mapView';
+
+interface MapContainerProps {
+  readonly neighborhoods: NeighborhoodGeoDataReducerState['neighborhoodGeoData'];
+  readonly blocks: BlockGeoDataReducerState['blockGeoData'];
+}
+
+const MapContainer: React.FC<MapContainerProps> = ({
+  neighborhoods,
+  blocks,
+}) => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getMapGeoData());
+  }, [dispatch]);
+
+  return (
+    <>
+      {blocks.kind === AsyncRequestKinds.Completed &&
+        neighborhoods.kind === AsyncRequestKinds.Completed && (
+          <MapView
+            blocks={blocks.result}
+            neighborhoods={neighborhoods.result}
+          />
+        )}
+    </>
+  );
+};
+
+const mapStateToProps = (state: C4CState): MapContainerProps => {
+  return {
+    neighborhoods: state.mapGeoDataState.neighborhoodGeoData,
+    blocks: state.mapGeoDataState.blockGeoData,
+  };
+};
```

Comment:
Typically it's better to keep connections to Redux in a container, letting components simply take props. Is there another map container already where this ducks folder could be added?

Reply:
These routes have to be used on each page with a map, so it would have to be on both the landing page and reservations. If it makes more sense to have the redux connections on both of those pages then we can do that